.github/workflows/checklist-score.yml

```
@@ -0,0 +1,147 @@
+
+          # Format status icon for categories based on completion
+          def get_status_icon(met, total):
+              if met == total:
+                  return "🟢"
+              elif met >= total * 0.75:
+                  return "🟡"
+              return "🔴"
+
+          cats = status["categories"]
+          table = f"""| Category           | Met | Total | Status |
+|--------------------|-----|-------|--------|
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected comment or line break

(syntax-check)